### PR TITLE
Fix BUGS

### PR DESCRIPTION
**Pull request by zqbake**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ SCI is a program to identify sub-compartments from HiC data. SCI utilizes graph
 ## Installation 
 
 ```sh
-$ python setup.py
+$ python setup.py install
```
Comment (zqbake) [inline README.md:26]: Fix 1

File: README.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ $ cd ..
 
 3. run SCI using the following command
 ```sh
-$ python sci.py -n test -f Input_sample/SCI_input.txt -r 100000 -g chromosome_sizes/hg19.chrom.sizes -o both -s 1 -k 5
+$ python -m sci.sci -n test -f Input_sample/SCI_input.txt -r 100000 -g chromosome_sizes/hg19.chrom.sizes -o both -s 1 -k 5
```
Comment (zqbake) [inline README.md:96]: Fix 2

File: sci/hic.py [hunk 3]
```diff
@@ -66,11 +66,12 @@ def load_interaction_data(self, contact_file):
...
             (chrom1, start1, end1, chrom2,
                 start2, end2, count) = line.strip().split()
-            
+
             if count.split('.')[0].isdigit() and start1.isdigit() and start2.isdigit():
                 i = int(start1) / self.res
                 j = int(start2) / self.res
-
+            else:
+                continue
```
Comment (zqbake) [inline sci/hic.py:74]: Fix 3. if this if test failed, I and j will not be declared, thus causing the later reference of I and j raising Undefined local reference error.

File: scripts/hic2sci.sh [hunk 4]
```diff
@@ -8,7 +8,7 @@ for ((i=1;i<=22;i++)); do
 	for ((j=$((i))+1;j<=22;j++)); do 
 		c1=chr${i}
 		c2=chr${j}
-		java -jar ${JUICERTOOLS} dump observed GW_KR ${infile} chr${i} chr${j}  BP ${res} | awk -v c1=chr${i} -v c2=chr${j} -v res=${res} '{OFS="\t"}{print c1,$1,$1+res,c2,$2,$2+res,$3}' >> ${outfile};
+		java -jar ${JUICERTOOLS} dump observed KR ${infile} chr${i} chr${j}  BP ${res} | awk -v c1=chr${i} -v c2=chr${j} -v res=${res} '{OFS="\t"}{print c1,$1,$1+res,c2,$2,$2+res,$3}' >> ${outfile};
```
Comment (zqbake) [inline scripts/hic2sci.sh]: Fix4.

Comment (AshoorH) [reply scripts/hic2sci.sh]: Hi, 
SCI originally works with GenomeWide-KR normalized (GW_KR). Check Juicer tools addNorm command to add the normalization.

Thanks
Haitham

Comment (zqbake) [reply scripts/hic2sci.sh]: ok, thanks.

Comment (AshoorH) [reply scripts/hic2sci.sh]: I can merge other Fixes if you can reverse this change. 

Thanks
Haitham

**Pull request merged**